bin/run_imgserv.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script is to be run inside imgserv container ONLY
+#
+source /opt/lsst/software/stack/loadLSST.bash
+setup lsst_distrib;
+# Set home dir
+cd /home/lsst
+# Start redis server
+redis-server /app/rootfs/etc/redis/redis.conf
+# create the directory for results in /tmp
+mkdir /tmp/imageworker_results
+# Start celery worker in imgserv
+# Note: by default, celery will start 1 worker process per CPU core
+celery --detach -A lsst.dax.imgserv.jobqueue.imageworker worker \
```

Comment:
I do like this.  It's simple, and it gets everything started, and it's straightforward.  It's too simplistic for production, for example, maybe redis will die and we need to restart it, or celery might die and we might not restart it, etc, but it's more than enough to test everything out until we prove we want to go this way!

Reply:
I take that as a compliment - so thanks!  For deployment in the long run, we do have other options to consider, perhaps involving fundamental changes to how we package the imgserv separate modules, e.g. frontend and backend.

Reply:
That's definitely how it was meant :)